DevCycle.SDK.Server.Local/ConfigManager/EnvironmentConfigManager.cs

```diff
@@ -120,7 +120,7 @@ private void OnInitialized(DevCycleEventArgs e)
 
         private string GetConfigUrl()
         {
-            return localOptions.CdnSlug != "" ? localOptions.CdnSlug : $"/config/v1/server/{sdkKey}.json";
```

Comment:
You'll need to add some sort of validation check to ensure that we're not going to break customers that are using an override for the slug